Azure-Samples/Cognitive-Services-Voice-Assistant
# Revise LocalAppData config/model changes

## Purpose
Recent changes to support the use of configuration files from local, writeable folders caused several breaks that rendered the app inoperable:

* File copy on startup threw exceptions for existing files -- including on the normal path
* Unnecessary recopy wiped applied settings on every startup
* Default config data didn't include needed information to make the keyword work

This change makes a range of adjustments to restore app functionality and align it with future changes.

## Does this introduce a breaking change?
[ ] Yes
[X] No

## Pull Request Type
```
[X] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test, What to Check

1. Clean uninstall/reinstall (no local folder)
2. Run the app, verify good startup
3. Verify config used/loaded from writeable location
4. Verify config changes persist on app restart
5. Verify 'contoso' keyword works

## Other Information
This change also introduces a level of structure to the config.json file.
